.travis.yml

```diff
@@ -0,0 +1,32 @@
+language: php
+
+matrix:
+    include:
+        - php: 7.1
+        - php: 7.0
+        - php: 5.6
+        - php: 5.5
+          env: REMOVE_TWITEROAUTH=true
+        - php: 5.4
```

Comment:
Having Debian 7 in active LTS does not means PHP is still maintained: security fix are not back ported. Using PHP 5.4 on a server is not secure anymore (even on Debian 7).

Dropping the PHP 5.4 and 5.5 support could only make less maintenance work for you guys. People that need the PHP 5.4 (or 5.5) support could still use the older release.